helm/config/log_format.conf

```diff
@@ -6,42 +6,51 @@ log_format main escape=json
 '{'
     '"Timestamp":$nanosec,'
     '"Attributes":{'
-        '"mapcolonies.time_local":"$time_local",'
+        '"mapcolonies.time_local": "$time_local",'
         {{ if .Values.authorization.enabled }}
-        '"mapcolonies.http.auth.token.client_name":"$jwt_payload_sub",'
+        '"mapcolonies.http.auth.token.client_name": "$jwt_payload_sub",'
         {{ end }}
-        '"http.request.method":"$request_method",'
-        '"http.request.header.referer":"$http_referer",'
-        '"http.request.body.size":"$content_length",'
-        '"http.response.body.size":"$body_bytes_sent",'
-        '"http.response.header.x_forwarded_for":"$proxy_add_x_forwarded_for",'
-        '"http.response.status_code":"$status",'
-        '"user_agent.original":"$http_user_agent",'
-        '"network.protocol":"$server_protocol",'
-        '"mapcolonies.request_time":"$request_time",'
-        '"mapcolonies.http.upstream_connect_time":"$upstream_connect_time",'
-        '"mapcolonies.http.upstream_response_time":"$upstream_response_time",'
-        '"mapcolonies.http.upstream_addr":"$upstream_addr",'
-        '"mapcolonies.http.upstream_status_code":"$upstream_status",'
-        '"mapcolonies.http.upstream_cache_status":"$upstream_cache_status",'
-        '"mapcolonies.server":"$hostname",'
-        '"server.address":"$host",'
-        '"server.port":"$server_port",'
-        '"client.address":"$remote_addr",'
-        '"client.port":"$remote_port",'
-        '"url.scheme":"$scheme",'
-        '"url.path":"$uri",'
-        '"url.full":"$request_uri"'
+        '"http.request.method": "$request_method",'
+        '"http.request.header.host": "$host",'
+        '"http.request.header.referer": "$http_referer",'
+        '"http.request.header.content-type": "$content_type",'
+        '"http.request.header.content-length": "$content_length",'
+        '"http.request.body.size": $request_length,'
+        '"http.request.size": $bytes_sent,'
+        '"http.response.body.size": $body_bytes_sent,'
+        '"http.response.header.x_forwarded_for": "$proxy_add_x_forwarded_for",'
+        '"http.response.status_code": "$status",'
+        '"user_agent.original": "$http_user_agent",'
+        '"network.protocol.name": "$otel_network_protocol_name",'
+        '"network.protocol.version": "$otel_network_protocol_version",'
+        '"mapcolonies.request_time": "$request_time",'
+        '"mapcolonies.http.upstream_addr": "$upstream_addr",'
+        '"mapcolonies.http.upstream_status_code": "$upstream_status",'
+        '"mapcolonies.http.upstream_connect_time": "$upstream_connect_time",'
+        '"mapcolonies.http.upstream_response_time": "$upstream_response_time",'
+        '"mapcolonies.http.upstream_response_length": $upstream_response_length,'
```

Comment:
what happens if the upstream returns 204? i remember it doesnt return a value and then the json is broken

Reply:
I'll put it in quotes.